docs/3_guides/9_aligned_cli.md

```diff
@@ -0,0 +1,118 @@
+# Aligned CLI
+
+The Aligned CLI serves as a interface for users to interact with and retrieve information from Alinged Layer. This document serves as a reference for commands of the Aligned CLI.
+
+## Installation:
+
+1. Download and install Aligned from the Aligned github repo `https://github.com/yetanotherco/aligned_layer`:
+
+```bash
+curl -L https://raw.githubusercontent.com/yetanotherco/aligned_layer/main/batcher/aligned/install_aligned.sh | bash
+```
+
+2. Run the ```source``` command that should appear in the shell.
+
+```bash
+aligned
+```
+
+## Commands:
```

Comment:
I would also add `--help` usage, for example:

## Help

To see the available commands run:
```bash
aligned --help 
```

To see the usage of a command run:
```bash
aligned [COMMAND] --help
```